src/potions/variable_meat_potion.ts

```diff
@@ -0,0 +1,147 @@
+import "core-js/modules/es.object.from-entries";
+import {
+  availableAmount,
+  Effect,
+  effectModifier,
+  haveEffect,
+  historicalAge,
+  historicalPrice,
+  Item,
+  mallPrice,
+  numericModifier,
+  print,
+  retrievePrice,
+  use,
+} from "kolmafia";
+import { maxBy } from "libram";
+import { acquire } from "../acquire";
+import { baseMeat, retrieveUntradeablePrice, turnsToNC } from "../lib";
+import { estimatedGarboTurns } from "../turns";
+import { globalOptions } from "../config";
+
+function triangleNumber(b: number, a = 0) {
```

Comment:
@Pantocyclus I think we should have more informative variable names here; what is the `a` parameter used for? why is `b` the lead variable?

Reply:
This function returns the sum of integers from `a` to `b` inclusive, with `a=0` by default
(It seems less intuitive to sum from `b` to `a`, setting `b=0` by default)